lib/components/map/point-popup.tsx

```diff
@@ -56,22 +77,33 @@ function MapPopup({
       closeOnClick
       latitude={mapPopupLocation.lat}
       longitude={mapPopupLocation.lon}
-      onClose={() => setMapPopupLocation({ location: null })}
+      onClose={clearMapPopupLocation}
+      // Override inline style supplied by react-map-gl to accommodate long "plan a trip" translations.
+      style={{ maxWidth: '260px', width: '260px' }}
     >
-      <PopupContainer>
+      <PopupTitleWrapper>
         <PopupTitle>
           {typeof popupName === 'string' && popupName.split(',').length > 3
             ? popupName.split(',').splice(0, 3).join(',')
             : popupName}
         </PopupTitle>
-        <div>
-          <FromToLocationPicker
-            label
-            location={{ ...mapPopupLocation, name: popupName }}
-            setLocation={onSetLocationFromPopup}
-          />
-        </div>
-      </PopupContainer>
+        <ZoomButton
+          // className="pull-right"
```

Comment:
Is this comment still needed?

Reply:
No, good catch, removed in https://github.com/opentripplanner/otp-react-redux/pull/667/commits/9c07af0c666936e4a65cb7328f9a6bca456d842e.